ndohyep/templatetags/ndohyep_tags.py

```diff
@@ -16,4 +18,15 @@ def is_in_group(user, group_name):
     return user.groups.filter(name__exact=group_name).exists()
 
 
+def handle_markdown(value):
+    md = markdown(
+        value,
+        [
+            'markdown.extensions.fenced_code',
+            'codehilite',
+        ]
+    )
+    return mark_safe(md)
```

Comment:
@miltontony there's a slight problem in that this code is adding `<p></p>` tags around each list item. Any idea how we can stop it doing that?